framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -2369,6 +2374,9 @@ private static void log(String format, Object... args) {
    *     Java type
    */
   public final boolean isRelevant(TypeMirror tm) {
+    if (tm == alwaysRelevantTM) {
```

Comment:
This is now the only use of `alwaysRelevantTM`.  I think we should just delete it.